test/unit/regression/SNYK-JS-AXIOS-7361793.js

```diff
@@ -36,10 +39,43 @@ describe('Server-Side Request Forgery (SSRF)', () => {
         // Malicious payload is as below.
         const userId = '/localhost:' + String(BAD_PORT);
 
-        const response = await ssrfAxios.get(`/${userId}`);
-        assert.strictEqual(response.data, 'good');
-        assert.strictEqual(response.config.baseURL, 'http://localhost:' + String(GOOD_PORT));
-        assert.strictEqual(response.config.url, '//localhost:' + String(BAD_PORT));
-        assert.strictEqual(response.request.res.responseUrl, 'http://localhost:' + String(GOOD_PORT) + '/localhost:' + String(BAD_PORT));
+        try {
+            await ssrfAxios.get(`/${userId}`);
+        } catch (error) {
+            assert.strictEqual(error.message, 'Invalid URL');
```

Comment:
```suggestion
            assert.strictEqual(error.message, 'Invalid URL:' + userId);
```

Reply:
It fails if I change `'Invalid URL:' + userId` what environment is failing on that?

Reply:
never mind, fixed that